src/navigation/index.tsx

```diff
@@ -34,9 +35,34 @@ export default function RootNavigation() {
 
   const authContext = useMemo(() => getAuthContext(dispatch), []);
 
+  const [activeStack, setActiveStack] = useState(null);
+
+  useEffect(() => {
```

Comment:
if possible can you see if you can consolidate this if and try logic

Reply:
inner one most if check can be ternary and can also check out && for short circuiting of conditional logic